yarn.lock

```diff
@@ -8707,6 +8707,13 @@ react-element-to-jsx-string@^14.3.4:
     is-plain-object "5.0.0"
     react-is "17.0.2"
 
+react-error-boundary@^4.0.3:
```

Comment:
Thank you for running yarn, I must've missed doing that when reviewing your previous branch errorHandle